### PR TITLE
Update GSD-2022-30190.json

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

File: 2022/30xxx/GSD-2022-30190.json [hunk 1]
```diff
@@ -1,7 +1,54 @@
 {
     "GSD": {
-        "alias": "CVE-2022-30190",
-        "id": "GSD-2022-30190"
```
Comment (athix) [inline 2022/30xxx/GSD-2022-30190.json:4]: The ID/Alias seem to be removed with no replacement?

**Review by athix — CHANGES_REQUESTED** [review]
Looks like some data was accidentally deleted and not replaced.

File: 2022/30xxx/GSD-2022-30190.json (hunk 1 above)
Comment (athix) [reply 2022/30xxx/GSD-2022-30190.json:4]: Looks good other than this though, interesting checking out those references.

**Comment by kurtseifried** [comment]
Fixed

**Pull request merged**